calico-cloud/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,60 @@
+---
+description: Install NodeLocal DNSCache
+---
+
+# Use NodeLocal DNSCache in your cluster
+
+## Big picture
+
+Set up NodeLocal DNSCache to improve DNS lookup latency.
+
+## Before you begin
+
+### Required
+
+Follow these [steps](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) to enable NodeLocal DNSCache connectivity.
+
+### Unsupported
+
+- OpenShift 
```

Comment:
Should we call out OpenShift as unsupported for this one feature when OpenShift is not supported on Calico Cloud?

Reply:
Good catch. This is the CE version of the doc. @tmjd But, I don't see that we call out non-support for OCP here: https://docs.tigera.io/calico-cloud/get-started/connect/requirements/system-requirements. Do I need to open a PR to add it?

Reply:
In addition to non-support for OCP in CC, I think we should also mention we don't support RKE and RKE2 in CC as well

Reply:
@bmckercher123 @tmjd @Glen-Tigera I removed the information about OpenShift and just added RKE and RKE2 in the `Unsupported` section.

Reply:
@girishshankaran We should remove all references to Unsupported (including RKE and RKE2) in this doc. The upshot of what Erik and Glen are saying is that CC does not support OCP, RKE, and RKE2 as platforms. Period. That needs to be called out in a separate PR to update CC system requirements. I will put up a PR for that.

Reply:
Done